packages/plugins/cjson/cjson.spec

```diff
@@ -28,17 +28,15 @@ developing applications that use cJSON.
 
 %build
 %cmake
-make %{?_smp_mflags}
```

Comment:
Note to self: this reverts 5a4edfdf077e67e68f3995c2d8169011ed064112

Reply:
as noted in the commit message, yes :)

Reply:
I wish GitHub made those more obvious in but GH and multi commits in PRs remains sub-optimal.